bundles/org.openhab.binding.modbus.e3dc/src/main/java/org/openhab/binding/modbus/e3dc/internal/dto/EmergencyBlock.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.e3dc.internal.dto;
+
+import static org.openhab.binding.modbus.e3dc.internal.modbus.E3DCModbusConstans.*;
+
+import java.nio.ByteBuffer;
+import java.util.BitSet;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.openhab.binding.modbus.e3dc.internal.modbus.Data;
+
+/**
+ * The {@link EmergencyBlock} Data object for E3DC Info Block
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class EmergencyBlock implements Data {
```

Comment:
NonnullByDefault

Reply:
Hmm, curious. Normally during maven build this highlighted each time. Added all NoNullBy Default annotations in dto package.

Reply:
Ah dto! Those can be without null annotations actually. They are not even checked in the static code analysis